server/src/main/java/keywhiz/service/resources/automation/v2/GroupResource.java

```diff
@@ -84,6 +84,7 @@ public class GroupResource {
   @Timed @ExceptionMetered
   @POST
   @Consumes(APPLICATION_JSON)
+  @AutomationAPIsLogging()
   public Response createGroup(@Auth AutomationClient automationClient,
```

Comment:
I think you can leave off the parentheses if your annotation doesn't require any values.